mempool/src/shared_mempool/coordinator.rs

```diff
@@ -171,11 +174,22 @@ pub(crate) async fn coordinator<V>(
                                 };
                             }
                             Event::RpcRequest((peer_id, msg, res_tx)) => {
-                                error!(
-                                    SecurityEvent::InvalidNetworkEventMempool,
-                                    message = msg,
-                                    peer_id = peer_id,
-                                );
+                                match network_id.network_id() {
+                                    NetworkId::Validator | NetworkId::Private(_) => {
+                                        error!(
+                                            SecurityEvent::InvalidNetworkEventMempool,
+                                            message = msg,
+                                            peer_id = peer_id,
+                                        );
+                                    }
+                                    _ => {
+                                        error!(
+                                            SecurityEvent::InvalidNetworkEventMempool,
+                                            peer_id = peer_id,
+                                        );
+                                    }
+                                }
```

Comment:
1) why is this an error? some remote peer sent us a flaky message, considering this can be easily triggered from the remote side, it seems a debug at best
2) the msg is interesting, I would consider using a sampling here if you really want to log this but I'm not convinced
3) perhaps we should just get a counter? because in general we won't be able to do anything with this except learn information